cron.json

```
@@ -11,6 +11,10 @@
     {
       "command": "8 7 * * * tools/send-orientations-reminders-emails.sh",
       "size": "S"
+    },
+    {
+      "command": "30 8 * * * tools/send_saved_searchs_notifications.sh",
```

```suggestion
      "command": "30 7 * * * tools/send_saved_searchs_notifications.sh",
```
7 c'est mieux, ça fait 8 ou 9h en heure locale.